brands/amenity/college.json

```diff
@@ -0,0 +1,20 @@
+{
+  "amenity/college|SAE Institute": {
+    "tags": {
+      "brand": "SAE Institute",
+      "brand:wikidata": "Q201438",
+      "brand:wikipedia": "en:SAE Institute",
+      "name": "SAE Institute",
+      "amenity": "college"
+    }
+  },
+  "amenity/college|Akademie Deutsche POP": {
+    "tags": {
+      "brand": "Akademie Deutsche POP",
+      "brand:wikidata": "Q413887",
+      "brand:wikipedia": "de:Akademie Deutsche POP",
+      "name": "SAE Institute",
```

Comment:
oh looks like this name is wrong

Reply:
That was wrong, but know it shouldn't be anymore.